tests/e2e/actions.go

```diff
@@ -262,110 +262,115 @@ func (tr Chain) submitTextProposal(
 }
 
 type UpdateConsumerChainAction struct {
-	Chain               ChainID
-	From                ValidatorID
-	ConsumerChain       ChainID
-	NewOwner            string
+	Chain              ChainID
+	From               ValidatorID
+	ConsumerChain      ChainID
+	NewOwner           string
+	InitParams         *InitializationParameters
+	PowerShapingParams *PowerShapingParameters
+}
+
+type InitializationParameters struct {
```

Comment:
I'm a bit confused about why the `InitializationParameters` and `PowerShapingParameters` types are redefined here and not imported from the provider types.

Reply:
two reasons for that decision:
- actions need to be independent from impelementation as they have to be able to deal with different versions
- some default values used in the tests espacially for initialization parameters which I would have to expose to the test otherwise.